crates/ruff_cli/src/cache.rs

```diff
@@ -98,10 +107,12 @@ impl Cache {
             path,
             package,
             new_files: Mutex::new(HashMap::new()),
+            last_seen_cache: SystemTime::UNIX_EPOCH.elapsed().unwrap().as_millis() as u64,
```

Comment:
Nit: Move the computation (and truncation) into a helper function to avoid repeating it (and scope the `allow` attribute) or create a `Cache::new_with_package(path, package)` method.